spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
+
+describe ReviewsController do 
+  describe "POST create" do
+    let(:video) {Fabricate(:video)}
+    context "with authenticated users" do
+      let(:current_user) { Fabricate(:user) }
+      before { session[:user_id] = current_user.id }
```

Comment:
Same here. Treat it sort of like taking a breath. You have one idea (and the let/before that belong to it), take a breath, and then a new idea.